app/models/incomeSourceDetails/BusinessDetailsModel.scala

```diff
@@ -28,10 +30,14 @@ case class BusinessDetailsModel(incomeSourceId: Option[String],
                                 cessation: Option[CessationModel],
                                 cashOrAccruals: Option[String] = None,
                                 address: Option[AddressModel] = None,
-                                latencyDetails: Option[LatencyDetails] = None) {
+                                latencyDetails: Option[LatencyDetails]) extends LatencyInformation {
```

Comment:
BusinessDetailsModel as far as I understand is part of response from the relevant API; so this is "given" and we have not much choice, so I am not sure if this is a good idea to manipulate with definition of this model;
from the other hand you are free to use this model to create any other required view model base on the requirement dictated in the UI logic etc